<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1517</width>
    <height>1031</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="search_button">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>검색</string>
    </property>
   </widget>
   <widget class="QListWidget" name="text_list">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>181</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QWebEngineView" name="search_view">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>1181</width>
      <height>1001</height>
     </rect>
    </property>
    <property name="url">
     <url>
      <string>about:blank</string>
     </url>
    </property>
   </widget>
   <widget class="QRadioButton" name="naver_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>네이버</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="youtube_btn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>유튜브</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="google_btn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>구글</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="search_text">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1517</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
